ports/stm32/boards/stm32f427_af.csv

```diff
@@ -0,0 +1 @@
+stm32f429_af.csv
```

Comment:
It looks like this is a symlink that was added as a file.  Symlinks should not be used.  Best to just remove this file, and reference `stm32f429_af.csv` in the `mpconfigboard.mk` file directly.

Reply:
I'll do it, even though I find it a bit misleading. [Datasheet](https://www.st.com/resource/en/datasheet/stm32f427vi.pdf) says (table 12 on page 75) that the alternate functions are exactly the same for the two microcontrollers. Shall we rename the old `stm32f429_af.csv` to something like `stm32f4279_af.csv`? At the moment there are 3 more boards relaying on the same file: `NUCLEO_F446RE`, `NUCLEO_F429ZI`, and `STM32F429DISC`.

Reply:
There are quite a few af.csv files that are used by multiple MCU series.  Let's leave the file name as-is and I suggest adding a comment in the mpconfigboard.mk file to mention that the f429 file is used because it's equivalent.

Reply:
OK. Done.